account_payment_mode/models/account_journal.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2016-2020 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    def _default_outbound_payment_methods(self):
+        all_out = self.env["account.payment.method"].search(
+            [("payment_type", "=", "outbound")]
+        )
+        return all_out
+
+    def _default_inbound_payment_methods(self):
+        all_in = self.env["account.payment.method"].search(
+            [("payment_type", "=", "inbound")]
+        )
+        return all_in
```

Comment:
fields to this method were removed 
do we need to migrate them or remove them as well?

Reply:
The methods are still present and used in account.journal:

https://github.com/odoo/odoo/blob/87e913b892c2518a716fceb247b57ab8c955a07f/addons/account/models/account_journal.py#L32-L36

https://github.com/odoo/odoo/blob/87e913b892c2518a716fceb247b57ab8c955a07f/addons/account/models/account_journal.py#L280-L302

so it totally makes sense to keep them in the module. 

(actually these were never method to compute a default value for a field on account.journal, but used to compute a default value for a payment being recorded on that journal)

Reply:
@i-vyshnevska fine for you?